h_periodic/h_beat.py

```diff
@@ -55,8 +55,8 @@
         "fill-annotations-pk": {
             "options": {"expires": 30},
             "task": "h.tasks.annotations.fill_pk_and_user_id",
-            "schedule": crontab(hour="9-12", minute="*/15"),
-            "kwargs": {"batch_size": 10000},
+            "schedule": crontab(hour="5-12", minute="*/10"),
+            "kwargs": {"batch_size": 15000},
```

Comment:
Some tasks have already gone over the time limit:

https://hypothesis.sentry.io/issues/4465586542/?referrer=slack&notification_uuid=adaafa01-d50f-4c25-b0ec-8cf3a50b4e10&alert_rule_id=7345746&alert_type=issue

I don't think we should increase, 4 min is plenty of time.